lambda/github_usage.py

```diff
@@ -177,7 +91,7 @@ def usage(message=None):
                 removed_usernames.append(username)
 
     for username in removed_usernames:
-        delete_ssm_param(f"{TOKEN_PREFIX}{username}")
+        config.delete_ssm_param(f"{TOKEN_PREFIX}{username}")
         del user_tokens[username]
         removed += 1
```

Comment:
It seems to me that the `removed` variable is not needed.
It's always incremented, no matter what happens, which means `removed` == `len(removed_usernames)`.

Reply:
Tweak line 104 from:

```python
"removed": removed,
```

```python
"removed": len(removed_usernames),
```

Reply:
See above.